pipelines/common/pom.xml

```diff
@@ -29,6 +29,7 @@
 
   <properties>
     <c3p0.version>0.9.5.5</c3p0.version>
+    <beam.version>2.44.0</beam.version>
```

Comment:
I think it is better to define this in the `pipelines/pom.xml` such that bot `batch` and `common` use the same version of Beam.

Reply:
Done